plans/agents.pp

```diff
@@ -4,13 +4,15 @@
   $server = get_targets('*').filter |$n| { $n.vars['role'] == 'pe' }
   # get agents ?
   $agents = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent/ }
+  $windows_agents = get_targets('*').filter |$n| { $n.vars['role'] == "agent_windows" }
 
 
   # install agents
   run_task('puppet_agent::install', $agents)
   # set the server 
   $server_string = $server[0].name
   run_task('puppet_conf', $agents, action => 'set', section => 'main', setting => 'server', value => "${server_string}")
+  run_command("powershell.exe -NoProfile -Nologo -Command 'Remove-Item -Path /ProgramData/PuppetLabs/puppet/etc/ssl -Force -Recurse'", $windows_agents)
```

Comment:
I have to be honest, I don't really know why nuking the ssl directory works, but it does :/

Reply:
yeah that's disappointing :( likely to do with how puppet_agent is working on windows.